analysisservice/src/test/java/nu/yona/server/analysis/service/AnalysisEngineService_determineRelevantGoalsAndroidTest.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
+ * 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.analysis.service;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import nu.yona.server.device.entities.DeviceAnonymized.OperatingSystem;
+import nu.yona.server.goals.service.GoalDto;
+
+public class AnalysisEngineService_determineRelevantGoalsAndroidTest extends AnalysisEngineService_determineRelevantGoalsTest
+{
+	@Override
+	protected OperatingSystem getOperatingSystem()
+	{
+		return OperatingSystem.ANDROID;
+	}
+
+	@Test
+	public void determineRelevantGoals_networkActivityOneOptionalUserGoal_userGoalOnIOs()
```

Comment:
Why is the operating system relevant in this test?

By the way, if it is relevant, it should read `determineRelevantGoals_networkActivityOneOptionalUserGoalOnAndroid_returnsEmpty`. Notice that it returned empty in the previous test?

Reply:
Good find! The test was wrong. On Android, the set of goals should be empty, as we are not measuring time based on network traffic there.

Even worse:: these tests didn't run at all because ``useJUnitPlatform`` wasn't added for the ``analysisengine`` project.